packages/docs-site/blog/2023/06/29/new-language-features.md

```diff
@@ -0,0 +1,302 @@
+<script setup>
+import BlogMeta from "../../../../src/components/BlogMeta.vue";
+</script>
+
+# What Have We Done to the Languages?
+
+<BlogMeta github="liangyiliang" date="2023-06-13" />
+
+I joined the Penrose team in July 2022, and worked on of a few language-related improvements to Penrose which have the potential of making Penrose more flexible.
```

Comment:
I might phrase this to be a bit stronger and more motivation-oriented. Maybe remove the phrase "have the potential of"; they _do_make Penrose more flexible. Then maybe write a brief overview of why your changes are useful, so people can have a high-level picture in their mind before they continue reading the rest of the details in the post.

Reply:
second this

Reply:
Done